wiki/Tournaments/CWC/2024/en.md

```diff
@@ -265,6 +305,15 @@ Saturday, 15 June 2024:
 | Russian Federation ::{ flag=RU }:: | 3 | **7** | ::{ flag=PE }:: **Peru** | [#1](https://osu.ppy.sh/community/matches/114303058) | [#1](https://www.twitch.tv/videos/2172850090) |
 | **Argentina** ::{ flag=AR }:: | **7** | 2 | ::{ flag=FI }:: Finland | [#1](https://osu.ppy.sh/community/matches/114304682) | [#1](https://www.twitch.tv/videos/2172930270) |
 
+Sunday, 16 June 2024:
+
+| Team A |  |  | Team B | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **Staff Red** | **7** | 6 | Staff Blue | [#1](https://osu.ppy.sh/community/matches/114315245) | [#1](https://www.twitch.tv/videos/2173679409) |
```

Comment:
this feels kind of out of place without an accompanying explanation that it's a showmatch

Reply:
I do agree but originally opted for this for archival purposes. An attempt at fixing this was added on a54ab4a